packages/client/src/processing/index.ts

```diff
@@ -68,7 +80,9 @@ export const processDictionary = (
 /**
  * Process a list of records for a single schema.
  *
- * Parse and then validate each record in the list.
+ * Parse and then validate each record in the list. If there are errors found during conversion,
+ * those errors will be returned and validation will be skipped. The final result will indicate if the
+ * data processing attempt was successful, or failed due to errors during parsing or validation.
```

Comment:
This description is right, but the params doesn't match what this function is expecting